Docs: Org, Team, and User Admin (#30756)
grafana/grafana: docs/sources/manage-users/server-admin/server-admin-manage-users.md

{{< docs/list >}}
  {{< docs/shared "manage-users/view-server-user-list-search.md" >}}
  1. In the User information section, click **Edit** next to the field that you want to change.
  1. Enter the new value and then click **Save**.
{{< /docs/list >}}

### Change the password on a user account

Users can change their own passwords, but Server Admins can change user passwords as well.

{{< docs/list >}}
{{< docs/shared "manage-users/view-server-user-list-search.md" >}}
1. In the User information section, click **Edit** next to the **Password** field.
1. Enter the new value and then click **Save**. Grafana requires a value at least four characters long in this field.
{{< /docs/list >}}

### Delete a user account

Permanently remove a user account from the server.

{{< docs/list >}}
{{< docs/shared "manage-users/view-server-user-list-search.md" >}}
1. Click **Delete User**.
1. Click **Delete user** to confirm the action.
{{< /docs/list >}}

### Enable or disable a user account

Temporarily turn on or off account access, but do not remove the account from the server.

#### Disable user account

> Prevent a user from logging in with this account, but do not delete the account. You might disable an account if a colleague goes on sabbatical.

Reviewer: ```suggestion
Prevents a user from logging in with this account, but does not delete the account. You may need to disable an account if a colleague goes on sabbatical.
```
